theseus/optimizer/sparse_linearization.py

```diff
@@ -20,7 +21,7 @@ def __init__(
         self,
         objective: Objective,
         ordering: Optional[VariableOrdering] = None,
-        **kwargs
+        **kwargs,
```

Comment:
Comma not needed.

Reply:
Comma at the end of multiline arguments is the standard in `black`.